device_access_fn.cpp

```diff
@@ -1701,16 +1705,16 @@ static DA_ReadResult readZclAttribute(const Resource *r, const ResourceItem *ite
 /*! A generic function to write ZCL attributes.
     The \p writeParameters is expected to contain one object (given in the device description file).
 
-    { "fn": "zcl", "ep": endpoint, "cl": clusterId, "at": attributeId, "dt": zclDataType, "mf": manufacturerCode, "eval": expression }
+    { "fn": "zcl:attr", "ep": endpoint, "cl": clusterId, "mf": manufacturerCode, "at": attributeId, "dt": zclDataType, "eval": expression }
 
     - endpoint: (optional) the destination endpoint
```

Comment:
Same as above, the endpoint isn't optional here. I think we should add checks to verify the parameters have the "ep".
This needs to be done in `readZclAttribute()` and `writeZclAttribute()` and `writeZclCommand()` since for `getZclParam()` it's valid that the "ep" is not present.

Reply:
Updated comment for `readZclAttribute()`, `sendZclCmd()`, `writeZclAttribute()`, `writeZclCommand()`.